kitebot/Jenkinsfile

```diff
@@ -163,16 +193,26 @@ void get_triggered_by() {
     return currentBuild.getBuildCauses()[0].shortDescription
 }
 
-void run_virt_test() {
+void run_virt_test(test_type) {
     sh label: 'wait until cloud-init done', script: """
         while true; do
             test -f /var/lib/cloud/instance/boot-finished && break
         done
     """
-    sh label: 'run ansible playbook', script: """
-        printenv
-        ./rhel-edge-virt-test.sh
-    """
+
+    if (test_type == 'ostree-ng') {
+        sh """
+            git clone -b rhel-8.4.0 https://github.com/osbuild/osbuild-composer.git
+            sudo mkdir -p /usr/libexec/osbuild-composer-test && sudo cp scripts/provision.sh /usr/libexec/osbuild-composer-test/provision.sh
```

Comment:
`sudo cp osbuild-composer/sudo cp scripts/provision.sh /usr/libexec/osbuild-composer-test`

Reply:
You mean change to:

sudo cp scripts/provision.sh /usr/libexec/osbuild-composer-test?

Reply:
`sudo cp osbuild-composer/scripts/provision.sh /usr/libexec/osbuild-composer-test`

Reply:
We already have a new version of provision.sh in this pull request to prepare the env in downstream, we don't need osbuild-composer/scripts/provision.sh .....

Reply:
ok, I see.